litho-it/src/test/java/com/facebook/litho/DynamicPropsTest.java

```diff
@@ -228,21 +221,40 @@ public Component build() {
           protected void setComponent(Component component) {}
         };
 
+    // We are using an old version of Robolectric which has a very limited
```

Comment:
We've recently migrated all our tests to the 4.x version, so I wonder if this is still true at all. Can you check if a plain LithoView would be enough, please?

Reply:
That is a great point! I have just used a plain LithoView and it worked beautifully. Thanks for the suggestion

Reply:
Thank you for fixing this, Christos !